docusaurus/docs-internals/editor-controls/data-controls/select.md

```diff
@@ -0,0 +1,415 @@
+| `onLoad`           | `VoidFunction`                                                                                                                                                                |                   -                    | A function that will be executed once the choices have been successfully loaded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
1. `VoidFunction` -> `Function`
2. A function that will be executed once the choices have been successfully loaded. -> A void function that will be executed once the async choices have been successfully loaded.
3. Mark `onLoad` as not required